dataformat-xml-dom/src/test/java/org/camunda/spin/impl/xml/dom/format/DomXmlDataFormatWriterTest.java

```diff
@@ -0,0 +1,121 @@
+package org.camunda.spin.impl.xml.dom.format;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.StringWriter;
+import javax.xml.transform.Templates;
+import org.camunda.spin.impl.util.SpinIoUtil;
+import org.camunda.spin.xml.SpinXmlElement;
+import org.junit.Test;
+import org.w3c.dom.Element;
+
+/**
+ * Test xml transformation in DomXmlDataFormatWriter
+ *
+ * @author Lars Uffmann, 2020-10-16
+ * @author Joern Muehlencord
+ */
+public class DomXmlDataFormatWriterTest {
```

Comment:
As mentioned above, the pretty print feature should be disabled by default. Otherwise, we will introduce some unexpected behavior to users. The tests here should be adjusted accordingly.

Reply:
I disagree with this assessment, especially since disabled pretty printing will now also create a differently formatted XML as it no longer used the indent feature.